docker/README.md

```diff
@@ -35,4 +35,10 @@ See also [targets to run tests](../docs/dev/how-to-write-and-run-tests.md#runnin
 | `make clean`              | Clean up your dev environment: removes locally bound folders, run `hdown` and `prune`. | Run `make dev` to recreate a fresh dev env afterwards.        |
 | `make import_sample_data` | Load sample data (~100 products) into the MongoDB database.                            |                                                               |
 | `make import_prod_data`   | Load latest prod data (~2M products, 1.7GB) into the MongoDB database.                 | Takes up to 10m. Not recommended for dev setups !             |
+| `make lint`      | Indent and reformat your code                         |
+
+PS : If you are having permission issues with `make lint` try writing the following commands :
```

Comment:
you can add footnotes, see https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes:


```suggestion
| `make lint`      | Indent and reformat your code[^lint]                       |

[^lint]: If you are having permission issues with `make lint` try writing the following commands :
```